pkg/agent/cniserver/pod_configuration.go

```diff
@@ -73,6 +74,8 @@ type podConfigurator struct {
 	// entityUpdates is a channel for notifying updates of local endpoints / entities (most notably Pod)
 	// to other components which may benefit from this information, i.e NetworkPolicyController.
 	entityUpdates chan<- types.EntityReference
+	// consumed by secondary network creation.
+	podInfoStore cnipodcache.CNIPodInfoStore
```

Comment:
I do not see podInfoStore is used in podConfigurator, and so we can move it to be a field of CNIServer?

Reply:
I don't think this comment from @jianjuns has been addressed,  but maybe you guys discussed it offline?

Reply:
Yes, @arunvelayutham said it is needed after refactoring the code to avoid passing CNIServer to podwatch.